src/test/commitments_test.cpp

```diff
@@ -481,19 +479,18 @@ TEST(TestCOMMs, TestCOMMCMGadget)
         ZERO);
 
     // cm should equal:
-    // cm = sha256(outer_k || 0^192 || value_v)
+    // cm = blake2sCompress(outer_k || 0^192 || value_v)
     //
     // outer_k:
     // 0xAF000000000000FF00000000000000FF00000000000000FF00000000000000FF '0^192
     // || value_v':
     // 0x0000000000000000000000000000000000000000000000002F0000000000000F
     //
-    // Note: This test vector has been generated by using the solidity sha256
-    // function (we want to make sure that we generate the same digests both
-    // on-chain and off-chain) Solidity version used: v0.5.0
+    // Note: This test vector has been generated by using the hashlib sha256
```

Comment:
Why is sha256 showing up here?

Reply:
Just updated with blake2s. I must have missed these notes.